tests/expected/reach/assert/reachable_fail/test.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-// kani-flags: --assertion-reach-checks --output-format regular --no-default-checks
+// kani-flags: --output-format regular --no-default-checks
```

Comment:
do we still need the output-format flag here? (and several more places below...)

(I'm also mildly surprised by the `--no-default-checks` but that seems unrelated: weren't we switching to "regular" because of assertion reachability and now both are default?)

Reply:
> do we still need the output-format flag here? (and several more places below...)
> 
No, we no longer need it. I'll remove it.

> (I'm also mildly surprised by the `--no-default-checks` but that seems unrelated: weren't we switching to "regular" because of assertion reachability and now both are default?)
>
The reason this was using `--no-default-checks` was to reduce the number of checks to make the expected tests less likely to pass due to some other check having "SUCCESS". Now that we support multi-line checks, `--no-default-checks` should no longer be needed. I'll remove it.